lib/index.js

```diff
@@ -1,16 +1,98 @@
 'use strict'
 
-var integration = require('@segment/analytics.js-integration')
-
+var integration = require('@segment/analytics.js-integration');
+var useHttps = require('use-https');
 /**
  * Expose `Wigzo` integration
  */
 
 var Wigzo = module.exports = integration('Wigzo')
+  .readyOnInitialize()
+  .global('wigzo')
+  .option('orgToken', '')
+  .option('features', [])
+  .tag('<script src="https://tracker.wigzopush.com/wigzo.js">')
+  .tag('https', '<link rel="manifest" href="/gcm_manifest.json">') /* This needs to be put inside <head>.Is this correct syntax ? */
+  .tag('https', '<script src="https://tracker.wigzopush.com/wigzopush_manager.js?orgtoken={{ orgToken }}">')
+  .tag('http', '<script src="https://tracker.wigzopush.com/gcm_http_subscribe.js?orgtoken={{ orgToken }}">');
 
 /**
  * Initialize Wigzo
  */
 
-Wigzo.prototype.initialize = function () {
-}
+Wigzo.prototype.initialize = function (){
+
+  window.wigzo.setIntegrationSource('Segment');
+
+  window.wigzo.init(settings.orgToken, settings.features);
+  var name = 'https';
+  if(!useHttps()){
+    name = 'http';
+    window.wigzo.httpGcmShowDialog = true;
+  } else {
+    window.wigzo.wigzoGcmAutoSubscribe = true;
+  }
+  
+  this.load(name, this.ready);
+};
+
+
+/**
+ * Loaded?
+ *
+ * @api private
+ * @return {boolean}
+ */
+
+Wigzo.prototype.loaded = function() {
+  return !!window.wigzo;
+};
+
+
+
+/**
+ * Identify.
+ *
+ * @api public
+ * @param {Identify} identify
+ */
+
+Wigzo.prototype.identify = function(identify) {
+  var traits = identify.traits();
+  var id = identify.userId();
+  if (id) window.wigzo.USERID = id;
+
+  window.wigzo.COOKIEID = identify.anonymousId;
+  if (traits) {
+    window.wigzo.identify({
```

Comment:
fyi -- `traits` here will be an empty object which is truthy. so if you want to make sure that you have email, phone, and fullname, you should just do:
```
var email = identify.email();
var phone = identify.phone();
var fullName = identify.name();
window.wigzo.identify({
    email: email ? email : '',
    phone: phone ? phone : '',
    fullName: name ? name : ''
});
```
Does this `wigzo.identify` function not handle situations where if you send undefined/null as email/phone/fullname? 

Or is it okay to not set it? that way we can write cleaner code:

```
var reject = require('reject');
var traits =  reject({
   email: identify.email(),
   phone: identify.phone(),
   fullName: identify.name()
});

window.wigzo.identify(traits);
```